specification/nginx/resource-manager/NGINX.NGINXPLUS/preview/2025-03-01-preview/swagger.json

```diff
@@ -0,0 +1,2850 @@
+          "readOnly": true
+        }
+      }
+    },
+    "NginxDeploymentWafPolicyApplyingStatus": {
+      "type": "object",
+      "title": "Nginx Deployment Waf Policy Applying Status",
+      "readOnly": true,
+      "properties": {
+        "code": {
+          "$ref": "#/definitions/NginxDeploymentWafPolicyApplyingStatusCode"
+        },
+        "displayStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A readable string of the current status, and sometimes have the reason for the current state."
+        },
+        "time": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The date and time in UTC the current applying status was set."
+        }
+      }
+    },
+    "NginxDeploymentWafPolicyApplyingStatusCode": {
+      "type": "string",
+      "example": "Applying",
+      "readOnly": true,
+      "description": "Machine readable code indicating the applying status code of a WAF Policy.",
+      "enum": [
+        "NotApplied",
+        "Applying",
+        "Succeeded",
+        "Failed",
+        "Removing"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "NginxDeploymentWafPolicyApplyingStatusCode",
+        "values": [
+          {
+            "value": "NotApplied",
+            "description": "The policy is not referenced in the nginx config and not applied."
+          },
+          {
+            "value": "Applying",
+            "description": "The policy is referenced in the nginx config and is applying."
+          },
+          {
+            "value": "Succeeded",
+            "description": "The policy is referenced in the nginx config and that config has been successfully applied."
+          },
+          {
+            "value": "Failed",
+            "description": "The policy is referenced in the nginx config and that config failed to apply."
+          },
+          {
+            "value": "Removing",
+            "description": "The policy is now not referenced in the nginx config and its being removed from the applied nginx config."
+          }
+        ]
+      }
+    },
+    "NginxDeploymentWafPolicyCompilingStatus": {
+      "type": "object",
+      "title": "Nginx Deployment Waf Policy Compiling Status",
+      "readOnly": true,
+      "properties": {
+        "code": {
+          "$ref": "#/definitions/NginxDeploymentWafPolicyCompilingStatusCode"
+        },
+        "displayStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A readable string of the current status, and sometimes have the reason for the current state. If the CompilingStatus is Failed the Display Status will be The waf Policy failed to compile."
+        },
+        "time": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The date and time the policy was compiled in UTC."
```

Comment:
>  "description": "The date and time the policy was compiled in UTC." [](http://example.com/codeflow?start=8&length=68)

is it the usual ISO format?
consider explaining in description,
and adding "format": "date-time" if so

Reply:
There's not specific time format the partner is following. The only thing that is certain is that the date and time will be provided in UTC time zone